client/dist/styles/styles.css

```diff
@@ -1,22 +1,31 @@
 /*
   Josh's Custom CSS Reset
   https://www.joshwcomeau.com/css/custom-css-reset/
+  meyerweb reset
+  http://meyerweb.com/eric/tools/css/reset/ 
 */
+
+/**************** BEGINNING OF CSS RESET **************** */
+
```

Comment:
I merged `reset.css` into `styles.css` as part of this pull request. The rules that modify stars are under the "beginning of stars" banner